04_numeronym/hsy3418/main.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fmt.Fprintln(out, numeronyms("accessibility", "Kubernetes", "abc"))
+}
+
+//numeronyms implements a function that returns a slice of numeronyms for its input strings
+func numeronyms(vals ...string) []string {
+
+	numeronymsList := vals
```

Comment:
You shouldn't name variables with "types" in their name, so in this case, do not add "List" to the variable name. Just name it for what it is. In this case, just `result` would be sufficient. `numeronyms` would also be ok too, but that is the name of the function, and it's best not to shadow names like that.